components/dashboard/tailwind.config.js

```diff
@@ -64,7 +64,16 @@ module.exports = {
                 "Segoe UI Symbol",
                 "Noto Color Emoji",
             ],
-            mono: ["JetBrains Mono", "SF Mono", "Monaco", "Inconsolata", "Fira Mono", "Droid Sans Mono", "Source Code Pro", "monospace"],
+            mono: [
+                "JetBrains Mono",
```

Comment:
I kept the same font as a monospaced font stack option for other use existing cases of the monospace fonts.